Source/DafnyCore/AST/Expressions/Expressions.cs

```diff
@@ -90,7 +90,7 @@ public abstract class Expression : TokenNode {
 
   public override string ToString() {
     try {
-      return Printer.ExprToString(this);
+      return Printer.ExprToString(DafnyOptions.DefaultImmutableOptions, this);
```

Comment:
Is there a reason to be sure that this doesn't require any non-default options? The same goes for other places where `DefaultImmutableOptions` is used.

Reply:
The only reason to be sure is that the build passes. I think `ToString()` should only be used for debugging purposes.